common/src/types/v0/message_bus/nexus.rs

```diff
@@ -200,6 +200,10 @@ impl NvmfControllerIdRange {
         const MAX_CONTROLLER_ID: u16 = 0xffef;
         MIN_CONTROLLER_ID ..= MAX_CONTROLLER_ID
     }
+    /// create a new NvmfControllerIdRange using provided range
+    pub fn new(start: u16, end: u16) -> Self {
```

Comment:
This should return an error, we should validate if the range is within 1..=0xffef ?